bloom-housing/bloom
# feat: adds ddog tracer

This PR addresses: there isn't a corresponding issue

## Description

This PR introduces Datadog's tracer agent to the backend API and public and partners sites. Following the documentation @ https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/dd_libraries/nodejs/

## How Can This Be Tested/Reviewed?

DataDog is already setup on core's backend in the prod environment. Once this is merged, we should be able to use the APM features within ddog and trace requests from the next apps to the backend.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
